invenio_github/assets/semantic-ui/js/invenio_github/index.js

```diff
@@ -0,0 +1,71 @@
+// This file is part of InvenioGithub
+// Copyright (C) 2023 CERN.
+//
+// Invenio Github is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+const sync_button = document.getElementById("sync_repos");
+if (sync_button) {
+  sync_button.addEventListener("click", function () {
+    const xhr = new XMLHttpRequest();
```

Comment:
What about using https://developer.mozilla.org/en-US/docs/Web/API/fetch . I think it would simplify the code a bit no?